src/main/java/address/prefs/UserPrefs.java

```diff
@@ -29,6 +31,12 @@ public void setSaveLocation(String saveLocation) {
         }
 
         writePrefToFile();
+
+        EventManager.getInstance().post(new SaveLocationChangedEvent(new File(saveLocation)));
```

Comment:
If we are to move this to a manager, we need to wrap `UserPrefs` methods into `ModelManager` methods and post the event there.